### ISSUE TITLE
custom vhost log

### ISSUE THREAD

**Issue by 532910** [debops/debops#1883]
While it's possible to completely customize global `access_log`/`error_log` with `nginx_http_access_log`/`nginx_http_error_log`:
https://github.com/debops/debops/blob/9f651b2276e74f0f4f0a9aa47ddc543661521452/ansible/roles/nginx/templates/etc/nginx/nginx.conf.j2#L63-L64

There is not way to do it for vhost (as at least `.log` is always appended):
https://github.com/debops/debops/blob/9f651b2276e74f0f4f0a9aa47ddc543661521452/ansible/roles/nginx/templates/etc/nginx/sites-available/default.conf.j2#L221-L230

This might be useful for passing logs to syslog for example.

**Comment by imrejonk** [comment]
Agreed, I've had the exact same issue some time ago. I worked around it by telling rsyslog to gather log lines from the log files in /var/log/nginx, but it would be nice to do something like [this](https://nginx.org/en/docs/syslog.html).

**Comment by imrejonk** [comment]
Agreed, I've had the exact same issue some time ago. I worked around it by telling rsyslog to gather log lines from the log files in /var/log/nginx, but it would be nice to do something like [this](https://nginx.org/en/docs/syslog.html).
